src/store/modules/user/actions.ts

```diff
@@ -70,6 +86,10 @@ const actions: ActionTree<UserState, RootState> = {
   async setFacility ({ commit }, payload) {
     commit(types.USER_CURRENT_FACILITY_UPDATED, payload.facility);
```

Comment:
We also need to fetch the facilityLocation whenever the current facility is changed.

Reply:
Updated code to fetch the facilityLocation whenever the current facility is changed.